packages/session/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@hono/session",
+  "version": "0.0.1",
+  "description": "Session middleware for Hono",
+  "type": "module",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsup ./src/index.ts ./src/cookies.ts ./src/helper/testing/index.ts",
+    "prepack": "yarn build",
+    "publint": "attw --pack --profile node16 && publint",
+    "typecheck": "tsc -b tsconfig.json",
+    "test": "vitest"
+  },
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      },
+      "require": {
+        "types": "./dist/index.d.cts",
+        "default": "./dist/index.cjs"
+      }
+    },
+    "./cookies": {
+      "import": {
+        "types": "./dist/cookies.d.ts",
+        "default": "./dist/cookies.js"
+      },
+      "require": {
+        "types": "./dist/cookies.d.cts",
+        "default": "./dist/cookies.cjs"
+      }
+    },
+    "./testing": {
+      "import": {
+        "types": "./dist/helper/testing/index.d.ts",
+        "default": "./dist/helper/testing/index.js"
+      },
+      "require": {
+        "types": "./dist/helper/testing/index.d.cts",
+        "default": "./dist/helper/testing/index.cjs"
+      }
+    }
+  },
+  "license": "MIT",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org",
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/honojs/middleware.git",
+    "directory": "packages/session"
+  },
+  "homepage": "https://github.com/honojs/middleware",
+  "peerDependencies": {
+    "hono": "*"
+  },
+  "dependencies": {
+    "@panva/hkdf": "^1.2.1",
```

Comment:
Oops, I looked at [the library](https://github.com/panva/hkdf), and then it was archived:

![CleanShot 2025-06-23 at 16 07 01@2x](https://github.com/user-attachments/assets/1ca161bf-5c7e-44f4-835d-b7120034ace2)

> This project is now archived. The functionality provided here is no longer necessary, as the Web Cryptography API is now widely supported across all major and modern JavaScript runtimes and you may just use the following

Should we use `Web Cryptography API` instead?

Reply:
Yes, we can. That's a very recent change 👀